test/web/namer_web_test.py

```diff
@@ -15,10 +16,7 @@
 from test.namer_watchdog_test import make_locations, new_ea, prepare
 from test.web.namer_web_pageobjects import FailedPage
 from namer.types import NamerConfig, default_config
-
-_browser = Chrome
-_options = ChromeOptions
-_manager = ChromeDriverManager
```

Comment:
Removed for now as different browsers will require a different function to configure the browser, let me know if we need this on you desktop now Dirty

Reply:
Don't like this global vars too, maybe do it like that

I don't have chrome, but edge is default for windows, so I'm used it

```python
from selenium.webdriver import Chrome as Browser, ChromeOptions as BrowserOptions
from selenium.webdriver.chrome.service import Service
from webdriver_manager.chrome import ChromeDriverManager as BrowserDriverManager
```

Reply:
I plan to use a single different functions for each supported browser  -- as each browser has it's own setup requirements.

Reply:
fixed.